contrib/babelfishpg_tsql/sql/sys_views.sql

```diff
@@ -2452,43 +2452,68 @@ create or replace view sys.dm_exec_connections
 CREATE OR REPLACE VIEW sys.configurations
 AS
 SELECT  configuration_id, 
-        name, 
-        value, 
-        minimum, 
-        maximum, 
-        value_in_use, 
-        description, 
+        name,
+        value,
+        minimum,
+        maximum,
+        value_in_use,
+        description,
         is_dynamic, 
         is_advanced 
-FROM sys.babelfish_configurations;
+FROM sys.babelfish_configurations
```

Comment:
we should use union with `pg_catalog.pg_settings in ('max_connection', 'babelfishpg_tds.tds_default_packet_size')` in predicate